frontend/src/templates/index.mustache

```diff
@@ -1,71 +0,0 @@
-<!doctype html>
-<html>
-<head>
-    <meta charset="utf-8" />
-    <link rel="shortcut icon" href="/static/images/favicon.ico">
-    <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
-    <link rel="stylesheet" href="/static/styles/main.css">
-
-    <meta name="defaultLanguage" content="en-US">
-    <meta name="availableLanguages" content="{{{ available_locales }}}">
-    <meta name="viewport" content="width=device-width">
-    <!-- Load en-US first as fallback for incomplete locales -->
-    <link rel="localization" href="/static/locales/en-US/app.ftl">
-    <link rel="localization" href="/static/locales/en-US/experiments.ftl">
-    <link rel="localization" href="/static/locales/{locale}/app.ftl">
-    <link rel="localization" href="/static/locales/{locale}/experiments.ftl">
-
-    <link rel="canonical" href="https://testpilot.firefox.com/{{{ canonical_path }}}">
-
-    <title data-l10n-id="pageTitleDefault">{{ meta_title }}</title>
-
-    <meta property="og:title" content="{{ meta_title }}" />
-    <meta name="twitter:title" content="{{ meta_title }}" />
-    <meta name="description" content="{{ meta_description }}" />
-    <meta property="og:description" content="{{ meta_description }}" />
-    <meta name="twitter:description" content="{{ meta_description }}" />
-    <meta name="twitter:card" content="summary" />
-    <meta property="og:image" content="{{{ image_facebook }}}" />
-    <meta name="twitter:image" content="{{{ image_twitter }}}" />
-</head>
-<body class="blue">
-  <div class="stars"></div>
-  <div id="page-container">
-    <noscript>
-      <div class="full-page-wrapper centered">
-        <div class="layout-wrapper layout-wrapper--column-center">
-          <div id="four-oh-four" class="modal delayed-fade-in">
-            <h1 data-l10n-id="noScriptHeading" class="title">Uh oh...</h1>
-            <div class="modal-content">
-              <p data-l10n-id="noScriptMessage">Test Pilot requires JavaScript.<br>Sorry about that.</p>
-            </div>
-            <div class="modal-actions">
-              <a data-l10n-id="noScriptLink" class="button default large" href="https://github.com/mozilla/testpilot/blob/master/docs/faq.md">Find out why</a>
-            </div>
-          </div>
-          <div class="copter">
-            <div class="copter__inner"></div>
-          </div>
-        </div>
-      </div>
-    </noscript>
-
-    <div class="full-page-wrapper centered overflow-hidden">
```

Comment:
FWIW, looks like these loader bars got left out of the new template - intentionally?